src/pseudopeople/noise_level.py

```diff
@@ -0,0 +1,71 @@
+from typing import TYPE_CHECKING
```

Comment:
This is copy/paste from noise_functions.py and is now an attribute of RowNoiseType do_not_respond.